freegs/__init__.py

```diff
@@ -40,3 +40,4 @@
 
 from .picard import solve
 
+from .dump import OutputFile
```

Comment:
Is it possible to make this a soft dependency, so that if h5py is not available then a warning is printed? OutputFile could fall back to a class which raises an error if actually used. Just thinking about cases where hdf5 isn't needed, and might be a pain to install

Reply:
`OutputFile` will now raise an exception if h5py is not available when you try to use it